tests/src/BuiltIn/AbstractBuiltInTest.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace Room11\Jeeves\Tests\BuiltIn;
+
+use Room11\Jeeves\Chat\Client\ChatClient;
+
+abstract class AbstractBuiltInTest extends \PHPUnit\Framework\TestCase
+{
+    protected $client;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->client = $this->getMockBuilder(ChatClient::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
@PeeHaa No, but it does help with SoC - https://github.com/Room-11/Jeeves/blob/0e2275f1ab42b9b6610294d015bbf21ae6c5dd58/tests/src/BuiltIn/Commands/AbstractCommandTest.php#L28 - thoughts?